packages/frontend/src/page/index.tsx

```diff
@@ -137,7 +145,7 @@ export const Component = ({
         );
     }, [chain, kpiToken]);
 
-    if (!kpiToken) {
+    if (!kpiToken || !kpiToken) {
```

Comment:
This is a double check

Reply:
Fixed in 269b3e4827e7970de94855a1c931891edfe13c21.